wp-content/themes/core/assets/css/src/theme/typography/_mixins.pcss

```diff
@@ -5,32 +5,202 @@
+	font-weight: 700;
+	letter-spacing: 0.1111px;
+}
+
+/* -------------------------------------------------------------------------
+ * Body Large
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body-large {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body-large);
+	line-height: 1.9;
+	font-weight: 400;
+}
+
+/* -------------------------------------------------------------------------
+ * Body
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body);
+	line-height: 2;
+	font-weight: 400;
+}
+
+/* -------------------------------------------------------------------------
+ * Body Small
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-body-small {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body-small);
+	line-height: 2;
+	font-weight: 400;
+	letter-spacing: 0.1px;
+}
+
+/* -------------------------------------------------------------------------
+ * Overline
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-overline {
+	font-family: var(--font-family-sans-serif);
+	font-size: 14px;
+	line-height: 2;
+	font-weight: 400;
+	letter-spacing: 1.75px;
+	text-transform: uppercase;
+}
+
+/* -------------------------------------------------------------------------
+ * Caption
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-caption {
+	@mixin t-body-small;
+}
+
+/* -------------------------------------------------------------------------
+ * Button
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-button {
+	font-family: var(--font-family-sans-serif);
+	font-size: var(--font-size-body);
+	line-height: 1.125;
+	font-weight: 700;
+	letter-spacing: 0.02em;
+}
+
+/* -------------------------------------------------------------------------
+ * Text Button/Call To Action
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-text-button {
+	font-family: var(--font-family-sans-serif);
+	font-size: 18px;
+	line-height: 1.5556;
+	font-weight: 700;
+	letter-spacing: 0.1111px;
+}
+
+/* -------------------------------------------------------------------------
+ * Input
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-input {
+	font-family: var(--font-family-sans-serif);
+	font-size: var(--font-size-body);
+	line-height: 1.5;
+	font-weight: 400;
+}
+
+/* -------------------------------------------------------------------------
+ * Label
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-label {
+	font-family: var(--font-family-sans-serif);
+	font-size: 14px;
+	line-height: 1.4;
+	font-weight: 400;
+}
```

Comment:
These probably belong in forms mixins?

Reply:
That's one way to think about it. However, design uses these typographic styles throughout the designs (such as the stylized list numbering) which means that don't just apply to forms.

Reply:
Sticking a pin in this for later, carry on, part of the contextual naming note.